docs/integrations/sources/aws-cloudtrail.md

```diff
@@ -0,0 +1,48 @@
+# AWS CloudTrail
+
+## Overview
+
+The AWS CloudTrail source supports both Full Refresh and Incremental syncs. You can choose if this connector will copy only the new or updated data, or all rows in the tables and columns you set up for replication, every time a sync is run.
+
+This Source Connector is based on a [Boto3 CloudTrail](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/cloudtrail.html).
```

Comment:
technical detail, not needed I think
```suggestion
```

Reply:
I think this is helpful for future reference and debugging.

Reply:
it's present in `setup.py` so a developer should be able to see it. Or are you saying a user might actually be interested in it? Fair enough. Apologies for the back and forth yaro, mind adding it back in?